cg/cli/workflow/rnafusion/base.py

```diff
@@ -258,33 +258,33 @@ def metrics_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
     except CgError as error:
         raise click.Abort() from error
 
-    if not analysis_api.trailblazer_api.is_latest_analysis_qc(case_id=case_id):
-        LOG.error("The analysis status must be in the QC step to be stored.")
-        raise click.Abort()
-    analysis_api.write_metrics_deliverables(case_id=case_id, dry_run=dry_run)
-    if dry_run:
-        LOG.info("Dry-run: QC metrics validation would be performed.")
-        return
-    try:
-        LOG.info("Validating QC metrics.")
-        analysis_api.validate_qc_metrics(case_id=case_id)
-    except MetricsQCError as error:
-        LOG.error(f"QC metrics failed for {case_id}")
-        analysis_api.trailblazer_api.set_analysis_status(
-            case_id=case_id, status=AnalysisStatus.FAILED
-        )
-        analysis_api.trailblazer_api.add_comment(case_id=case_id, comment=str(error))
-        raise click.Abort() from error
-    except CgError as error:
-        LOG.error(f"Could not create metrics deliverables file: {error}")
+    if analysis_api.trailblazer_api.is_latest_analysis_qc(case_id=case_id):
+        analysis_api.write_metrics_deliverables(case_id=case_id, dry_run=dry_run)
+        if dry_run:
+            LOG.info("Dry-run: QC metrics validation would be performed.")
+            return
```

Comment:
I would remove the `dry_run` from `write_metrics_deliverables` and move the `if dry_run:` statement outside the `if is_qc` block, which is already quite complex

Reply:
You can leave it if you feel it brings value to log the mock writing of metrics :)

Reply:
I changed it. I think its more readable